src/core/auth/decorators/roles.decorator.ts

```diff
@@ -0,0 +1,5 @@
+import { SetMetadata } from '@nestjs/common';
+import { UserRole } from '../../../constants';
```

Comment:
Rutas absolutas
```suggestion
import { UserRole } from 'constants';
```
o
```suggestion
import { UserRole } from 'src/constants';
```
o
```suggestion
import { UserRole } from '@app/constants';
```
Idk como manejen las rutas absolutas, pero debe ser absoluto